cmd/monaco/integrationtest/account/deploy-download_test.go

```diff
@@ -72,6 +74,9 @@ func TestIdempotenceOfDeployment(t *testing.T) {
 	for _, u := range deploy1st.Users {
 		assert.Contains(t, download1st.Users, u.Email.Value())
 	}
+	for _, deployedServiceUser := range deploy1st.ServiceUsers {
```

Comment:
We can remove this loop, because we are testing it already in the loop below

Reply:
That's a good question. More to the point, should the test stop here if this fails? Before the second deploy and then download? Here we are interested in did we manage to download what we think we deployed, where as the second time we ultimately want to check did we affect something by re-deploying.

Reply:
I would vote for `require`. Then I could leave the rest as it is 👍

Reply:
Ok, now I see - we probably shouldn't use require, otherwise delete won't run, which is cleaning the test up

Reply:
We could also `defer` the delete instead

Reply:
I was looking into that vs `t.Cleanup(...)`. We currently use `defer` though, so I think I'll go with that for now.